Resources/Prototypes/_Moffstation/PersonalItems/Items/Compilatron/Tania Buttersworth/stationblueprints.yml

```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: BaseItem
+  id: PersonalItemStationBlueprints
+  name: Station blueprints
+  description: A highly detailed schematic for a space station, complete with floor plan, wiring, pipe networks and other infrastructure. Clearly the proud work of a veteran station engineer.
+  suffix: PersonalItem
+  components:
+  - type: Sprite
+    sprite: Objects/Tools/blueprint.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Tools/blueprint.rsi
+    size: Small
+  - type: Tag
+    tags:
+    - Document
```

Comment:
Dyou know what the `Document` tag does functionally? I'm wondering if this would make it faxable or something.

Reply:
It does not make it faxable but it does make it possible to store in folders.